lib/mavenlink/client.rb

```diff
@@ -87,16 +91,16 @@ def connection_options
       }.freeze
     end
 
-    def parse_request(response)
-      if response.present?
-        parsed_response = JSON.parse(response)
+    def parse_request(body, status)
+      if body.present?
+        parsed_response = JSON.parse(body).merge("status" => status)
       else
         return
       end
 
       parsed_response.tap do
         Mavenlink.logger.whisper 'Received response:'
-        Mavenlink.logger.inspection response
+        Mavenlink.logger.inspection body
```

Comment:
This could be a secure information leak if these log messages are beamed off the box

Reply:
Agreed. That sounds like an existing issue with the gem. We'll definitely want to consider the secure data issue when we implement a separate logging service. That comes a bit later in the roadmap.